types/tailwindcss/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
tslint cannot add/disable any rules. remove these and fix the errors.

Reply:
I have removed such rules